parsl/tests/manual_tests/test_htex_worker_loss.py

```diff
@@ -0,0 +1,130 @@
+import argparse
```

Comment:
I am skeptical of tests that aren't plugged into CI-- they are of limited value if they are never run. We could consider re-designing this test to assert something, add the `@pytest.mark.local` decorator so that it is only run once, with the locally defined config, and move it to a pytest-accessible directory.

Reply:
This is not going into CI. To engineer the situation that triggers the failure condition is way too much work to try automating, so I'd rather defer on that but still have this test exist so that I may not have to redo this from scratch.